cmd/main.go

```diff
@@ -53,7 +52,7 @@ var (
 	useBaseBt     = flag.Bool("use_base_bt", true, "Use base bigtable cache")
 	baseTableName = flag.String("base_table", "", "Base cache Bigtable table.")
 	// Branch Bigtable Cache
-	useBranchBt = flag.Bool("use_branch_bt", false, "Use branch bigtable cache")
+	useBranchBt = flag.Bool("use_branch_bt", true, "Use branch bigtable cache")
```

Comment:
how do we catch such regressions?

Reply:
We should have a prober to fetch the few stat vars from branch cache and make sure the latest date is no more than N days.